wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
+
+class MACMatcher():
+    """
+        This class is using Organizationally Unique Identifiers (OUIs) 
+        in order to match MAC addresses of devices to its vendord  
+
+        See http://standards.ieee.org/faqs/OUI.html
+    """
+    def __init__(self, mac_prefix_file=MAC_PREFIX_FILE):
```

Comment:
Think of `macmatcher` as a generic module that can be used on different projects. `path` can be a variable in many cases.

What I don't like here is that we provide the path in the module per se. What we should do instead, is provide the path as an argument during the object initialization. Specifically, on `pywifiphisher.py`, we should do: `mac_matcher = macmatcher.MACMatcher(MAC_PREFIX_FILE)`.